packages/entities/entities-shared/src/components/entity-filter/EntityFilter.vue

```diff
@@ -292,7 +292,6 @@ const applyFields = (hideMenu = false) => {
   :deep(.k-input) {
     padding-bottom: 4px!important;
     padding-top: 4px!important;
-    width: auto;
```

Comment:
Can you check this against the latest Kongponents alpha? The input fields did nit have the correct box-sizing attribute

Reply:
Sure, so I tried this change with `@kong/kongponents@9.0.0-alpha.63`, and it looks good:
<img width="319" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/468d13cb-2629-4ce3-850e-471e8236a0f9">

The `input.k-input` has `width: 100%` and `box-sizing: border-box`:
https://github.com/Kong/kongponents/blob/664644f71cc7e2db131fe9830dbe89c09f1916e1/src/styles/mixins/_input-text.scss#L21
https://github.com/Kong/kongponents/blob/664644f71cc7e2db131fe9830dbe89c09f1916e1/src/styles/mixins/_input-text.scss#L9

Reply:
I was thinking with the latest your change wouldn't be needed here 😢

Reply:
Ah I see. I verified again with the latest Kongponents, and this change is still needed. With `width: auto` the `input.k-input` element will have a fixed width and is not responsive with its container.